src/listeners.js

```diff
@@ -0,0 +1,17 @@
+import { displayWeather } from './ui';
+
+function loadListeners() {
+  document.querySelector('.search-submit').addEventListener('click', () => {
+    const place = document.querySelector('.search').value;
```

Comment:
In the ui you call this location and I agree, that `location` fits better here as well. As well as in other places that you use word `place`